style.css

```diff
@@ -0,0 +1,146 @@
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo {
+  margin-right: auto;
+  align-items: center;
+  color: #6070ff;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.social {
+  list-style-type: none;
+  display: flex;
+}
+
+nav li {
+  list-style: none;
+  display: inline-block;
+  padding: 0 10px;
+  color: #344563;
+}
+
+.social li {
+  display: inline;
+  margin: 10px;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+}
+
+h1 {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+}
+
+.icon {
+  height: 5%;
+}
+
+p {
+  color: #344563;
+}
+
+header p {
+  max-width: 50%;
+}
+
+header img {
+  max-width: 10%;
+}
+
+.topimg {
+  display: flex;
+  justify-content: space-around;
+}
+
+.downimg {
+  display: flex;
+  justify-content: space-around;
+}
+
+.imgorder {
+  display: flex;
+  flex-direction: column;
+  align-items: first baseline;
+  max-width: 100%;
+  height: auto;
+}
+
+.imgorder2 {
+  width: 30%;
+  align-items: center;
+}
+
+.imgorder3 {
+  width: 90%;
+  align-items: flex-end;
+  margin: 20px;
+}
+
+.imgorder1 {
+  width: 50%;
+  align-items: flex-start;
+}
+
+.imgorder4 {
+  width: 40%;
+  align-items: flex-start;
+}
+
+.imgorder5 {
+  align-items: center;
+}
```

Comment:
This is a smart way to address the background issue mentioned by the previous reviewer ☝🏽 but there's an even smarter one. The one you're using now makes you work harder and will cause lots of trouble when the time comes to design the Desktop 🖥️ version. You can easily download the background image with all these frames from Figma (See how to do that [here](https://designlab.com/blog/how-to-export-images-from-figma/#:~:text=When%20you're%20designing%20with,SVG%2C%20or%20PDF%20image%20formats.)). You will then just need to add the `background-image` property on your headline `section`, although I think you should have a `section` wrapping everything below the `header` but as it wasn't mentioned, do not pay much attention to it. See how the images you added are pushing your `hx` below and adding unnecessary space between the `title` and the `toolbar` 👇🏽 

## Figma
![Screenshot 2023-02-18 at 08 38 13](https://user-images.githubusercontent.com/45739963/219868786-6190b6c8-6542-452a-a273-cf69ef8e33eb.png)


## Your design
![Screenshot 2023-02-18 at 08 37 51](https://user-images.githubusercontent.com/45739963/219868770-76504961-8a18-469f-b49c-3c9ddf924357.png)

The background-image CSS property allows you to then place the image behind any HTML element you wish. This could either be the whole page (by using the body selector in CSS which targets the <body> element in our HTML), or just a standalone specific part of the page such as a section element.  🤓